django/db/backends/base/schema.py

```diff
@@ -36,10 +36,18 @@ def _all_related_fields(model):
 def _related_non_m2m_objects(old_field, new_field):
     # Filter out m2m objects from reverse relations.
     # Return (old_relation, new_relation) tuples.
-    return zip(
+    first_related_fields = zip(
         (obj for obj in _all_related_fields(old_field.model) if _is_relevant_relation(obj, old_field)),
         (obj for obj in _all_related_fields(new_field.model) if _is_relevant_relation(obj, new_field)),
     )
+    related_fields = []
```

Comment:
It appears to me that, since this previously returned an iterator (`zip()`'s return value), you don't need to construct an intermediate list here and can simply `yield` the tuples directly.

Reply:
Good call, thanks 👍